design/sketch.webidl

```diff
@@ -413,23 +413,23 @@ enum GPUStencilOperation {
 };
 
 dictionary GPUStencilStateFaceDescriptor {
-    GPUCompareFunction compare;
-    GPUStencilOperation failOp;
-    GPUStencilOperation depthFailOp;
-    GPUStencilOperation passOp;
+    GPUCompareFunction compare = "less";
```

Comment:
Wouldn't `"always"` make sense here too, like for `depthCompare`?

Reply:
Ah yes, both [D3D12](https://docs.microsoft.com/en-us/windows/desktop/direct3d12/cd3dx12-depth-stencil-desc) and [Metal](https://developer.apple.com/documentation/metal/mtlstencildescriptor/1462455-stencilcomparefunction) use `always`.